src/shared/util.ts

```diff
@@ -16,3 +16,17 @@ export const endWithSlash = (s: string) => (s.endsWith('/') ? s : s + '/');
 
 export const getRandomString = () =>
   crypto.getRandomValues(new Uint32Array(1))[0].toString(32);
+
+/**
+ * If the given URL object is a Skypack URL, perform an in-place update that
+ * switches from optimized mode to raw mode.
```

Comment:
is there a Skypack bug you can reference?

Reply:
Added link to our bug.